src/utils/auth/signInSignUp.js

```diff
@@ -0,0 +1,17 @@
+import makeRequest from '/src/utils/services/makeRequest';
+
+export const signInSignUp = async (email, password, signURL, changeState) => {
+  const response = await makeRequest(signURL, {
+    method: 'POST',
+    body: JSON.stringify({
+      email,
+      password,
+      returnSecureToken: true,
+    }),
+  });
+  const data = await response.json();
+  if (response.status !== 200) {
```

Comment:
Better would be something like `response.status >=400`, because only 400+ codes are used for reporting errors.